chunks/tool-tailscale/Dockerfile

```diff
@@ -9,6 +9,10 @@ ENV TRIGGER_REBUILD=1
 RUN curl -fsSL https://pkgs.tailscale.com/stable/ubuntu/focal.gpg | sudo apt-key add - \
```

Comment:
Should we change the way we store apt keys in this PR itself?

https://github.com/gitpod-io/workspace-images/blob/main/CONTRIBUTING.md#adding-a-chunk

```
# fetch keyring over https connection and unpack it using gpg's --dearmor option
curl -fsSL https://apt.my-secure.org/my-unofficial-repo.gpg.key | sudo gpg --dearmor -o /usr/share/keyrings/my-unofficial-repo.gpg.key
# and then add them to a apt key sources list.
echo "deb [signed-by=/usr/share/keyrings/my-unofficial-repo.gpg.key] http://apt.my-secure.org/focal/ \
my-unofficial-repo-toolchain main" | sudo tee /etc/apt/sources.list.d/my-unofficial-repo.list > /dev/null
```

Reply:
That makes sense - @princerachit please feel free to push a commit or open a new PR